src/main/java/dude/task/ToDo.java

```diff
@@ -0,0 +1,36 @@
+package dude.task;
+
+/**
```

Comment:
Very thorough and detailed comments even for a small and simple class. YOURE TOO GOOD